app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    friends = []
+    for person in people:
+        friends.append(Person(person["name"], person["age"]))
+
+    for person in people:
+        if person.get("wife"):
+            Person.people[person["name"]].wife\
+                = Person.people[person["wife"]]
+        if person.get("husband"):
+            Person.people[person["name"]].husband\
+                = Person.people[person["husband"]]
```

Comment:
try not to use line breakers, u can make separate variable for this Person.people[person["name"]] to avoid line breaking, or split line by [] brackets